dbbuild/marker.tsv

```diff
@@ -25,7 +26,7 @@ mh01USC-1pD	MHDBM-8098d345	GRCh38	chr1	106227453,106227475,106227487	3.1574	0.37
 mh01KK-211	MHDBM-6adc547b	GRCh38	chr1	161985865,161985967,161986013	2.2742	0.1706	0.1164	ALFRED
 mh01AT-03	MHDBM-be82bac2	GRCh38	chr1	161985967,161985984,161986013	2.7654	0.1143	0.0071	ISFG2019:P597
 mh01USC-1qA	MHDBM-f218f0ad	GRCh38	chr1	167157730,167157749	2.5003	0.0976	0.0695	10.1016/j.fsigen.2019.102213
-mh01NH-03	MHDBM-e7a95c5e	GRCh38	chr1	184807944,184807966,184808042	1.9085	0.1987	0.1725	10.1016/j.legalmed.2015.06.003
+mh01NH-03	MHDBM-e7a95c5e	GRCh38	chr1	184807944,184807966,184808042	1.9086	0.1987	0.1725	10.1016/j.legalmed.2015.06.003
```

Comment:
A couple of these rounding artifacts pop up every once in a while depending on numpy version. 🤨